<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewMapDialog</class>
 <widget class="QDialog" name="NewMapDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Map Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>229</width>
        <height>306</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="verticalSpacing">
        <number>10</number>
       </property>
       <item row="3" column="1">
        <widget class="QLabel" name="label_MapIDError">
         <property name="visible">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0)</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="NoScrollComboBox" name="comboBox_LayoutID">
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::InsertPolicy::NoInsert</enum>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="label_LayoutIDError">
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0)</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_NameError">
         <property name="visible">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0)</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEdit_MapID">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The constant that will be used to refer to this map. It cannot be the same as any other existing map, and it must start with the specified prefix.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_MapID">
         <property name="text">
          <string>Map ID</string>
         </property>
        </widget>
       </item>
       <item row="13" column="0" colspan="2">
        <widget class="QWidget" name="widget_HeaderData" native="true">
         <layout class="QVBoxLayout" name="layout_HeaderData">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_CanFlyTo">
         <property name="text">
          <string>Can Fly To</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_Name">
         <property name="text">
          <string>Map Name</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="NoScrollComboBox" name="comboBox_Group">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The name of the group this map will be added to.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::InsertPolicy::NoInsert</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_Name">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The name of the new map. The name cannot be the same as any other existing map.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QCheckBox" name="checkBox_CanFlyTo">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If checked, a Heal Location will be added to this map automatically.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="label_GroupError">
         <property name="visible">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 0, 0)</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="0" colspan="2">
        <widget class="NewLayoutForm" name="newLayoutForm" native="true"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_Group">
         <property name="text">
          <string>Map Group</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_LayoutID">
         <property name="text">
          <string>Layout ID</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_GenericError">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok|QDialogButtonBox::StandardButton::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NewLayoutForm</class>
   <extends>QWidget</extends>
   <header>newlayoutform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NoScrollComboBox</class>
   <extends>QComboBox</extends>
   <header>noscrollcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
